src/api/api.cc

```diff
@@ -11862,7 +11875,6 @@ void recordreplay::SetRecordingOrReplaying(void* handle) {
   // Log disabled features.
   if (gRecordReplayDisabledFeatures) {
     for (const std::string& feature : *gRecordReplayDisabledFeatures) {
-      fprintf(stderr, "RecordReplayDisabledFeature %s\n", feature.c_str());
```

Comment:
Let's not remove this printf, it is helpful when we get recording logs from customers and are diagnosing problems that happen while recording.

Reply:
Ok. I removed it because thought it was an old comment that I added and forgot to remove earlier